.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
```

Comment:
This only has one OS, so this whole file could be simplified, like the others.

Reply:
Since the default os is not Linux, should I still include the os name in the job name?

Reply:
It should be exactly like the Ubuntu ones, just `macos-latest` instead.